src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -303,6 +308,158 @@ public override bool IsSealed
             }
         }
 
+        public override int Arity
+        {
+            get
+            {
+                return _underlyingType.Arity;
+            }
+        }
+
+        public override ImmutableArray<TypeParameterSymbol> TypeParameters
+        {
+            get
+            {
+                return _underlyingType.TypeParameters;
+            }
+        }
+
+        internal override ImmutableArray<ImmutableArray<CustomModifier>> TypeArgumentsCustomModifiers
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override bool HasTypeArgumentsCustomModifiers
+        {
+            get
+            {
+                return _underlyingType.HasTypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override ImmutableArray<TypeSymbol> TypeArgumentsNoUseSiteDiagnostics
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsNoUseSiteDiagnostics;
+            }
+        }
+
+        public override NamedTypeSymbol ConstructedFrom
+        {
+            get
+            {
+                return _underlyingType.ConstructedFrom;
+            }
+        }
+
+        public override bool MightContainExtensionMethods
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override string Name
+        {
+            get
+            {
+                return string.Empty;
+            }
+        }
+
+        internal override bool MangleName
+        {
+            get
+            {
+                return _underlyingType.MangleName;
+            }
+        }
+
+        public override IEnumerable<string> MemberNames
+        {
+            get
+            {
+                return _fields.Select(f => f.Name);
+            }
+        }
+
+        internal override bool HasSpecialName
+        {
+            get
+            {
+                return _underlyingType.HasSpecialName;
```

Comment:
made it false.
I think many emit related properties are not supposed to be used since tuple types are not going to be emitted. It is hard to tell whether properties like IsSerializable should defer to underlying or not. It probably still should even though it may not be serializable regardless.